### PR TITLE
Add instructions on how to format multiple variables

### PR DESCRIPTION
**Pull request by marcusolsson**
**What this PR does / why we need it**:

Adds instructions on how to set default variable format for multiple variables.

Thanks @yesoreyeram for noticing this gap in our docs!

### CODE REVIEW

File: docs/sources/developers/plugins/add-support-for-variables.md [hunk 1]
```diff
@@ -51,14 +51,34 @@ For data sources, you need to use the [getTemplateSrv]({{< relref "../../package
...
 
      const data = makeDbQuery(query);
 
      return { data };
    }
    ```
 
+## Format multiple variables
+
+If the user has selected multiple values, the value of the interpolated variable depends on the [variable format](https://grafana.com/docs/grafana/latest/variables/advanced-variable-format-options/).
```
Comment (oddlittlebird) [inline docs/sources/developers/plugins/add-support-for-variables.md:64]:
```suggestion
If the user selects multiple values, then the value of the interpolated variable depends on the [variable format](https://grafana.com/docs/grafana/latest/variables/advanced-variable-format-options/).
```

Comment (oddlittlebird) [reply docs/sources/developers/plugins/add-support-for-variables.md:64]: Unclear what you mean with "multiple values." Do you mean multiple variable values? A variable with multiple values? Multi-value variables? Selected how? In the dropdown? Please clarify.

File: docs/sources/developers/plugins/add-support-for-variables.md [hunk 2]
```diff
@@ -51,14 +51,34 @@ For data sources, you need to use the [getTemplateSrv]({{< relref "../../package
...
+
+A data source can define the default format option when no format is specified, by adding a third argument to the interpolation function.
```
Comment (oddlittlebird) [inline docs/sources/developers/plugins/add-support-for-variables.md:66]:
```suggestion
A data source can define the default format option when no format is specified by adding a third argument to the interpolation function.
```

**Review by oddlittlebird — CHANGES_REQUESTED** [review]

File: docs/sources/developers/plugins/add-support-for-variables.md (hunk 1 above)
Comment (marcusolsson) [reply docs/sources/developers/plugins/add-support-for-variables.md:64]: All of what you said is correct :) Maybe "When a user selects multiple values for variable"?

**Review by dprokop — APPROVED** [review]

File: docs/sources/developers/plugins/add-support-for-variables.md (hunk 1 above)
Comment (oddlittlebird) [reply docs/sources/developers/plugins/add-support-for-variables.md]: Yes, that makes more sense.

**Review by oddlittlebird — APPROVED** [review]
Good job, Marcus!

**Pull request merged**